templates/test/ci/prow-aks-multi-tenancy/patches/aks-agentpool0.yaml

```diff
@@ -3,4 +3,5 @@ kind: AzureManagedMachinePool
 metadata:
   name: "agentpool0"
 spec:
-  maxPods: 24
+  maxPods: 32
```

Comment:
let's remove this and wait for #1971 to merge

Reply:
rebased